infrastructure/environments/dplplat01/configuration/mariadb-servers/values.yaml

```diff
@@ -0,0 +1,11 @@
+namespace: mariadb-servers
+name: mariadb-10.6-01
+
+password: VerySecurePassword
+rootPassword: SomeSecurePassword
```

Comment:
I would add the secrets outside of the helm chart, so you can commit the helm chart and values to the gitops repo.

Reply:
Of course

Reply:
I see this, which fixes it: 295887e65d1e1ed28abd6a5143350e25d754cb54